BrushUpCode/src/Coffee.java

```diff
@@ -0,0 +1,272 @@
+import java.util.Scanner;
+
+public class Coffee {
+    private int coffeeBeans = 120;
+    private int water = 400;
+    private int milk = 540;
+    private int cups = 9;
+    private int dollar = 550;
+
+    private int espressoWater = 250;
+    private int espressoCoffee = 16;
+    private int espressoMoney = 4;
+
+    int latteWater = 350;
+    int latteCoffee = 20;
+    int latteMilk = 75;
+    int latteMoney = 7;
+
+    int cappuccinoWater = 200;
+    int cappuccinoMilk = 100;
+    int cappuccinoCoffee = 12;
+    int cappuccinoMoney = 6;
+
+    State coffeeMachineState = State.NONE;
+    Scanner enter = new Scanner(System.in);
+
+    /*constructor*/
+    public Coffee() {
```

Comment:
@maria-salmasian do you need public constructor or it can be private ?

Reply:
it cannot be private, because I use 2 classes Coffee and CoffeeDemo. I need an object of Coffee in CoffeeDemo. Private will not allow me to make one.

Reply:
@maria-salmasian okay but the public mean available from everywhere, but your classes are in the same package, you can restrict accessibility a little bit

Reply:
changed it to protected